recipe/meta.yaml

```diff
@@ -7,7 +7,7 @@ package:
 source:
   fn: dask-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/d/dask/dask-{{ version }}.tar.gz
-  sha256: 566054b493d63c15732f2a640382b21e861571d61639f59341bc7695a9be138e
+  sha256: 1fa7e1809e7c50c5505297b03e6ee161e0695df73b4c4051b5b1c8fd670bf5d7
 
 build:
   number: 0
```

Comment:
Could you please bump the build number so we can rebuild without squashing the existing package? For the existing build 0, we'll apply a repodata hotfix.

Reply:
Yes, makes sense, I needed to make another PR because this branch is not my fork!